homeassistant/components/google_sheets/application_credentials.py

```diff
@@ -1,13 +1,15 @@
 """application_credentials platform for Google Sheets."""
+import oauth2client
+
 from homeassistant.components.application_credentials import AuthorizationServer
 from homeassistant.core import HomeAssistant
 
 
 async def async_get_authorization_server(hass: HomeAssistant) -> AuthorizationServer:
     """Return authorization server."""
     return AuthorizationServer(
-        "https://accounts.google.com/o/oauth2/v2/auth",
-        "https://oauth2.googleapis.com/token",
+        oauth2client.GOOGLE_AUTH_URI,
```

Comment:
What's going on here? We removed this yesterday. The PR description doesn't explain it.